src/test/java/com/spotifytelegramdeliverymessage/SpotifyTelegramDeliveryMessageApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.spotifytelegramdeliverymessage;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
```

Comment:
write a full test or remove it completely

Reply:
deleted so far